lib/signaling/v2/activespeakersignaling.js

```diff
@@ -0,0 +1,61 @@
+'use strict';
+
+const { EventEmitter } = require('events');
+
+/**
+ * @property {?Track.SID} loudestParticipantSid
+ * @emits ActiveSpeakerSignaling#updated
+ */
+class ActiveSpeakerSignaling extends EventEmitter {
```

Comment:
The more I write classes like these—EventEmitters with a single event, e.g. "updated"—the more I'm feeling like we should go ahead and adopt Observable. The [zen-observable](https://github.com/zenparsing/zen-observable) is lightweight and pretty solid.